src/main/java/core/AbstractGameState.java

```diff
@@ -106,41 +93,72 @@ void reset(long seed) {
         reset();
```

Comment:
Wasn't the reset() method on GS supposed to be removed?

Reply:
No - it's the abstract _reset() method I've removed; I don't see anything wrong with the reset() method as it stands, as game implementors need know nothing about it.